mephisto/core/operator.py

```diff
@@ -336,6 +327,24 @@ def shutdown(self, skip_input=True):
             self.supervisor.shutdown()
             self._run_tracker_thread.join()
 
+    def validate_and_run_config_wrap(
```

Comment:
Since this function will be user-facing (I see it in script files) I find that the `_wrap` portion isn't too descriptive. It doesn't really have much utility for the user to know that this is simply a wrapper around something else. The immediate next questions are: what does it wrap around? why do we need a wrapper? etc.

The comment description here is pretty descriptive for me to answer those questions.

Based on some conventions I've seen in the past, names such as these might be more descriptive:

`validate_and_run_config_or_die` for the underlying method, and just `validate_and_run_config` for this one.
Or `safely_validate_and_run_config` for this one, I've seen this usage in cases where a method returns a default value in cases where it would normally throw an exception.